### PR TITLE
Updates for node10 and Cleanups

### PR DESCRIPTION
**Pull request by rejas**
Cleaned up package info
Updated all dependencies
Added node10 to travis tests and removed node4

### CODE REVIEW

**Comment by rejas** [comment]
hi @tail could you spare some time to review this?

**Comment by rejas** [comment]
Anybody here?

**Comment by tecfu** [comment]
Going to fork and republish as grunt-mocha-redux because I need a package to be published with the dependencies updated.

**Comment by rejas** [comment]
thx @tecfu will you add this PR too or should I create one in your new repo again? and are you planning on updating it yourself in the future?

**Comment by thekashifmalik** [comment]
Sorry for the slow turn-around; the disqus team has been through some changes and it's hard to keep up with all our open source project.

@rejas reviewing now.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - 4
   - 6
   - 8
-  - 9
+  - 10
```
Comment (thekashifmalik) [inline .travis.yml]: Are you removing Node 9 since it's not an LTS release?

File: Gruntfile.js [hunk 2]
```diff
@@ -65,11 +75,13 @@ module.exports = function(grunt) {
...
             grep: 'food'
           },
+          growlOnFail: false,
+          growlOnSuccess: false,
 
           reporter: 'Nyan',
 
           // URLs passed through as options
-          urls: ['http://localhost:' + port + '/example/test/test2.html'],
+          urls: ['http://localhost:' + port + '/example/test/test2.html']
```
Comment (thekashifmalik) [inline Gruntfile.js]: Generally prefer the trailing comma since it leads to fewer lines changed if you add a new item to the parent dict.

**Review by thekashifmalik — APPROVED** [review]
Changes seem fine.

File: .travis.yml (hunk 1 above)
Comment (rejas) [reply .travis.yml]: yeah, its not supported anymore. I#d add 11 if you want to?

File: Gruntfile.js (hunk 2 above)
Comment (rejas) [reply Gruntfile.js]: Changed again

File: .travis.yml (hunk 1 above)
Comment (thekashifmalik) [reply .travis.yml]: Nah it's fine.

**Review by thekashifmalik — APPROVED** [review]
looks fine. will cut a release shortly

**Comment by thekashifmalik** [comment]
@rejas here you go: https://www.npmjs.com/package/grunt-mocha/v/1.2.0

**Comment by rejas** [comment]
Thx for the merge and release, really appreciate it! And good luck with those team changes!

**Pull request merged**